File: Examples/Objective-C/Remote Control/Remote Control/AdminViewTabBarController.m
//
//  Copyright © 2020 Bridgefy Inc. All rights reserved.
//

#import "AdminViewTabBarController.h"

@interface AdminViewTabBarController () <UITabBarDelegate>

@end

@implementation AdminViewTabBarController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.title = @"Admin panel";
}

- (void)sendObject:(id)object sender:(id)sender {
    [self.mvc sendObject:object sender:sender];
}

@end

File: Examples/Swift/Remote Control/Remote Control/AdminViewTabBarController.swift
//
//  Copyright © 2020 Bridgefy Inc. All rights reserved.
//

import UIKit

class AdminViewTabBarController: UITabBarController {
    
    public var mvc: MainViewController?

    override func viewDidLoad() {
        super.viewDidLoad()
        
        title = "Admin panel"
    }
    
    public func send(object: Any?, sender: Any) {
        if (mvc != nil) {
            mvc!.send(object: object, sender: sender);
        }
    }
    
}
